tests/rptest/tests/partition_balancer_test.py

```diff
@@ -584,9 +584,9 @@ def get_node2partition_count():
         with self.NodeStopper(self) as ns:
             for n in range(7):
                 node = self.redpanda.nodes[n % len(self.redpanda.nodes)]
-                ns.make_unavailable(node, wait_for_previous_node=True)
                 try:
                     admin_fuzz.pause()
+                    ns.make_unavailable(node, wait_for_previous_node=True)
```

Comment:
right, the intent behind pausing is to avoid interleaving fuzzing with fault injection (test resumes fuzzing after it confirms that the fault is injected) but because the pause was put after make_unavailable there could be an interleaving